purchase_order_line_schedule_stock/models/purchase_order_line.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2021 ForgeFlow, S.L.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+from odoo import models
+
+
+class PurchaseOrderLine(models.Model):
+
+    _inherit = "purchase.order.line"
+
+    def _prepare_stock_moves(self, picking):
```

Comment:
I still only get one stock move per POL, I think it might be good to use purchase_delivery_split_date for this.

Reply:
Non prio feedback.